tools/db_bench_tool.cc

```diff
@@ -410,6 +410,9 @@ DEFINE_double(read_random_exp_range, 0.0,
 
 DEFINE_bool(histogram, false, "Print histogram of operation timings");
 
+DEFINE_bool(confidence_interval_95, false,
```

Comment:
Maybe call it `confidence_interval_only` for printing 95% CI upper and lower bounds only, rather than average ± 95% error.